Files/warranty_dell.py

```diff
@@ -77,114 +137,101 @@ def process_result(self, result, purchases):
         global full_serials
         data = {}
 
-        if 'AssetWarrantyResponse' in result:
-            for item in result['AssetWarrantyResponse']:
-                try:
-                    warranties = item['AssetEntitlementData']
-                    asset = item['AssetHeaderData']
-                    product = item['ProductHeaderData']
-                except IndexError:
-                    if self.debug:
-                        try:
-                            msg = str(result['InvalidFormatAssets']['BadAssets'])
-                            if msg:
-                                print '\t\t[-] Error: Bad asset: %s' % msg
-                        except Exception as e:
-                            print e
+        for item in result:
+            try:
+                warranties = item['entitlements']
+            except IndexError:
+                if self.debug:
+                    try:
+                        msg = str(result['InvalidFormatAssets']['BadAssets'])
+                        if msg:
+                            print '\t\t[-] Error: Bad asset: %s' % msg
+                    except Exception as e:
+                        print e
 
+            else:
+                # saw this order number code, did not see this in the response with test devices,
+                # but leaving it here in case there is a product that does return this information
+                # if self.order_no == 'vendor':
```

Comment:
Do we need these 2 commented out code lines.

Reply:
i'll remove, I didn't see this value in the response. If we need to add them back in will do so